habitat_baselines/config/rearrange/ddppo_open_fridge.yaml

```diff
@@ -23,7 +23,7 @@ NUM_CHECKPOINTS: 10
 FORCE_TORCH_SINGLE_THREADED: True
 EVAL_KEYS_TO_INCLUDE_IN_NAME: ['reward', 'force', 'success']
 
-BASE_TASK_CONFIG_PATH: configs/tasks/rearrange/open_fridge.yaml
+BASE_TASK_CONFIG_PATH: habitat/config/tasks/rearrange/open_fridge.yaml
```

Comment:
If we want to make this pip installable perhaps we should have a special symbol etc that points to the configs stored in the site / userwide installation? Otherwise, what is the motivation for moving them to within the habitat folder?

Reply:
After all, if the configs are user / install location dependent, it would be a lot worse in practice, right? BUT if we have a special symbol / env variable / etc that points to the correct location, then that would be fine.

Reply:
I was thinking about this. 
At first the reason I was doing it was because #705 started doing it, the reason is because if we want the users to pip install the package, they need to have some default configurations that are inside the package. For example, when doing 

```
env = gym.make("HabitatOpenCab-v0")
```

We need to have the config for the open cab task in the sitepackage.
**BUT** if the user wants to create/edit the tasks for their use cases, then the recommended way should be to clone the repo and edit the configs from there.
(We should not have the user edit configs in sitepackage.)

Note that this work is only to make habitat-lab pip installable. I do not think habitat-baselines should be pip installable (at least not for now).
What do you think?

Reply:
@vincentpierre Exactly, but how does a user distinguish a default config from the file "HabitatOpenCab-v0". That functionality is the thing that is incomplete from this PR. if add that, then I agree this is a very useful and excellent PR.

Reply:
I made it such that if you pip install in non-editable mode (like with git+) the configs are saved inside the package. So the user has access to the default gym configs when installing this way. If the user changes their minds and re-does the installation with editable mode, the configs from the repo will be used. Does that address your concern?

Reply:
habitat installation is in PYTHONPATH. That means that it will be the default configs directory. If you want to change it and use, user should copy the default config to their config dir and use it for launching the training or clone the repo.